theseus/utils/examples/pose_graph/relative_pose_error.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Optional, Tuple, Union
+
+import torch
+
+import theseus as th
+
+
+class RelativePoseError(th.CostFunction):
```

Comment:
[Between](https://github.com/facebookresearch/theseus/blob/main/theseus/embodied/measurements/between.py)?

Reply:
A little different. `Between` is `measurement^-1 * v0^-1 * v1` while `RelativePoseError` is `v1^-1*v0*measurement`. Also, it seems that the `jacobian` of `Between` is incorrect.

Reply:
Same question regarding the incorrect jacobian.

Reply:
Just saw the comments in the chat thread, we can continue the conversation there.